src/test/resources/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqScoreSpark/expected_paired.txt

```diff
@@ -1,766 +1,766 @@
 tax_id	taxonomy	type	name	kingdom	score	score_normalized	reads	unambiguous	reference_length
-10240	root|Viruses|dsDNA_viruses,_no_RNA_stage|Poxviridae	family	Poxviridae	Viruses	56.0	4.643449419568822	56	56	0
```

Comment:
No change needed now but I suggest that in the future you format output float numbers to a fix maximum number of decimals to be shown (e.g 4 most significant numbers) with ```String.format```. On the one hand is not visually appealing but also it might mean that some tribal changes in the code (e.g change in the order of float point operations) could break the test due to a very unimportant decimal value flip.

Reply:
Yes I've had this problem and currently in tests check if the value is equal within 6 digits. Truncating would be preferable, though.